src/libraries/Microsoft.Extensions.Caching.Memory/src/CacheEntry.CacheEntryTokens.cs

```diff
@@ -127,7 +127,7 @@ private static void InvokeCallbacks(CacheEntry entry)
 
                     try
                     {
-                        registration.EvictionCallback?.Invoke(entry.Key, entry.Value, entry.EvictionReason, registration.State);
+                        registration.EvictionCallback?.Invoke(entry.Key, entry.Value!, entry.EvictionReason, registration.State!);
```

Comment:
?? Why is this necessary?

Reply:
When I removed the reference project I got this nullability warning which I had to suppress.

Reply:
That's not good. That means we probably have something wrong here.

cc @maxkoshevoi

Reply:
To be fair, this was the only warning that was emitted when deleting all Extensions reference projects :)

Reply:
Good catch! There seems to be a CI issue where is doesn't warn on `ref`  / `src` nullability mismatch.
Created a PR to fix `src` nullability: https://github.com/dotnet/runtime/pull/65804